lightning.profile

```diff
@@ -23,6 +23,11 @@ function lightning_install_tasks() {
   $tasks['lightning_set_logo'] = [];
   $tasks['lightning_alter_frontpage_view'] = [];
 
+  $tasks[Drupal\lightning\Form\TelemetryOptInForm::class] = [
```

Comment:
By `::class` syntax, I meant we should have a `use` statement at the top of the file, then this array key can be a much more readable `TelemetryOptInForm::class`. :)